src/vs/editor/contrib/hover/browser/contentHover.ts

```diff
@@ -758,6 +756,13 @@ export class ContentHoverWidget extends ResizableContentWidget {
 			this._hover.containerDomNode.focus();
 		}
 		hoverData.colorPicker?.layout();
+
+		// The aria label overrides the label, so if we add to it, add the contents of the hover
+		let accessibleViewHint = getHoverAccessibleViewHint(this._configurationService.getValue('accessibility.verbosity.hover') === true && this._accessibilityService.isScreenReaderOptimized(), this._keybindingService.lookupKeybinding('editor.action.accessibleView')?.getAriaLabel() ?? '');
+		if (accessibleViewHint) {
+			accessibleViewHint = ', ' + accessibleViewHint;
+		}
+		this._hover.contentsDomNode.ariaLabel = accessibleViewHint ? this._hover.contentsDomNode.textContent + accessibleViewHint : '';
```

Comment:
Preferably we wouldn't touch it if accessible View Hint is not set, I think setting it to '' will clear the actual label and then not be available for other assistive tech

Reply:
it doesn't do anything wrong (I tested it), but also fine to go with what you suggested.